app-modules/engagement/src/Models/Engagement.php

```diff
@@ -223,6 +224,11 @@ public function getMergeData(): array
             'student full name' => $this->recipient->getAttribute($this->recipient->displayNameKey()),
             'student email' => ProspectStudentRefactor::active() ? $this->recipient?->primaryEmailAddress?->address : $this->recipient->getAttribute($this->recipient->displayEmailKey()),
             'student preferred name' => $this->recipient->getAttribute($this->recipient->displayPreferredNameKey()),
+            'user first name' => (new Parser())->parse($this->user->name)->getFirstname(),
+            'user full name' => $this->user->name,
+            'user job title' => $this->user->job_title ?? 'staff member',
```

Comment:
I think unless Product gave direction to do so. Let's not have the null coalescing here. We can think of some ways to maybe warn the User for values that may potentially be blank. But let's not put a stand in for now.